SUMMARY.md

```diff
@@ -97,6 +97,7 @@
   - [UI Shell](libs/application/ui-shell/README.md)
 - [Auth API Lib](libs/auth-api-lib/README.md)
 - [Content Search Indexer](libs/content-search-indexer/README.md)
+- [Content search toolkit](libs/content-search-toolkit/README.md)
```

Comment:
To be consistent it should be `Content Search Toolkit`, no?

Reply:
Yes, I updated manually to keep consistency.
We may need to iterate `yarn gitbook` to handle this since it's the prefered method to update `SUMMARY.md`?
@jeremybarbet

Reply:
I was thinking to create a custom schematic for nx so it uses a custom README.md template, where the title would be uppercase and with a `gitbook-ignore` by default when the readme is "empty". 

If we only change the `SUMMARY.md` we will end up having dash titles also on the page on gitbook